e2e/scaffolding/ethereum.ts

```diff
@@ -0,0 +1,101 @@
+import { KeyringPair } from '@polkadot/keyring/types';
+import { encodeAddress, ethereumEncode } from '@polkadot/util-crypto';
+import { hexToU8a, u8aToHex } from '@polkadot/util';
+import type { Signer } from '@polkadot/types/types';
+import { SignerResult } from '@polkadot/types/types';
+import { secp256k1 } from '@noble/curves/secp256k1';
+import { Keyring } from '@polkadot/api';
+import { Keypair } from '@polkadot/util-crypto/types';
+import { Address20MultiAddress } from './helpers';
+
+export function getUnifiedAddress(pair: KeyringPair): string {
+  if ('ethereum' === pair.type) {
+    const etheAddressHex = ethereumEncode(pair.publicKey);
+    return getConvertedEthereumAccount(etheAddressHex);
+  }
+  if (pair.type === 'ecdsa') {
+    throw new Error(`ecdsa type is not supported!`);
+  }
+  return pair.address;
+}
```

Comment:
this might be required at quite a few places at some point/utility, should be documented?